projects/archiver/types/object.fromentries.d.ts

```diff
@@ -0,0 +1,12 @@
+declare function index<K extends PropertyKey, T = any>(
```

Comment:
Would be nice to have this everywhere!

Reply:
[it](https://github.com/microsoft/TypeScript/issues/31393) [would](https://node.green/#ES2019-features--Object-fromEntries)

Reply:
I've moved to using ramda's frompairs